symbolic-base/src/ZkFold/Symbolic/Class.hs

```diff
@@ -71,6 +72,10 @@ class ( HApplicative c, Package c, HNFData c
 embed :: (Symbolic c, Functor f) => f (BaseField c) -> c f
 embed cs = fromCircuitF hunit (\_ -> return (fromConstant <$> cs))
 
+-- | Embeds the witness value(s) into generic context @c@. Inverse of 'witnessF'.
```

Comment:
I think it's worth mentioning here that the resulting variables are going to be unconstrained